lectures/global-entrans-tuwien/index.rst

```diff
@@ -0,0 +1,62 @@
+Global Energy Transitions and Climate Policy @ TU Wien
+============================================
+
+.. figure:: ../../../../_static/tuwien_logo.png
+   :width: 240px
+   :align: right
+
+This course is part of the Master's Degree Program in `Environmental Technology & International Affairs`_ 
+held in Technical University Vienna (TU Wien) since spring 2021.
```

Comment:
```suggestion
held at the Technical University Vienna (TU Wien) since spring 2021.
```

Reply:
Thanks @danielhuppmann for the quick review and spotting these. I corrected the typos and the URLs.